ui/servsettings.py

```diff
@@ -288,3 +297,31 @@ async def get_content(self):
             description=await self.get_inline_rolling_desc()
         )
         return {"embed": embed}
+
+class _MiscellaneousSettingsUI(ServerSettingsMenuBase):
+    # ==== ui ====
+    @disnake.ui.button(label='Toggle DDB Campaign Message', style=disnake.ButtonStyle.primary, row=1)
+    async def toggle_campaign_cta(self, _: disnake.ui.Button, interaction: disnake.Interaction):
+        self.settings.show_campaign_cta = not self.settings.show_campaign_cta
+        await self.commit_settings()
+        await self.refresh_content(interaction)
+
+    @disnake.ui.button(label='Back', style=disnake.ButtonStyle.grey, row=4)
+    async def back(self, _: disnake.ui.Button, interaction: disnake.Interaction):
+        await self.defer_to(ServerSettingsUI, interaction)
+
+    # ==== content ====
+    async def get_content(self):
+        embed = disnake.Embed(
+            title=f"Server Settings ({self.guild.name}) / Miscellaneous Settings",
+            colour=disnake.Colour.blurple(),
+            description="These settings affect less significant parts of Avrae."
```

Comment:
```suggestion
```
The wording here feels a little awkward; we can probably just drop the description and let the settings speak for themselves